### PR TITLE
Blogpost: Onboarding

### PR DESCRIPTION
**Pull request by Chimney42**
This blogpost will update our users on our new onboarding experience:
- Docs
- Tutorial
- Demo

### CODE REVIEW

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+image_caption: "Brown and red windmills in front of blue sky. Photo by Harry Cunningham @harry.digital from <a href='https://www.pexels.com/photo/brown-and-red-wind-mill-3405489/'>Pexels</a>"
+description: "making old things new"
+tags:
+  - tutorial
+  - tilt
+  - devx
+---
+Hallo Tilters,
+
+Big announcement incoming: we have redesigned the Tilt experience! 🥳
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:17]: "Does this mean Tilt has a new UI?"

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+description: "making old things new"
+tags:
+  - tutorial
+  - tilt
+  - devx
+---
+Hallo Tilters,
+
+Big announcement incoming: we have redesigned the Tilt experience! 🥳  
+Today we’re taking a look at our onboarding. Most of you who are reading this are probably already acquainted with Tilt— don’t worry, we’ve got something coming for you, too.
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:18]:
```suggestion
Today we’re taking a look at our onboarding tutorial. Most of you who are reading this are probably already acquainted with Tilt— and don’t worry, we’ve got something coming for you, too, but:
```

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Everyone knows: you can only make one first impression, better make it count! So with that in mind, we wanted to create an experience that is quick, easy and fun, even if you’re starting from zero. That goes for both yourself and the friends you want to introduce Tilt to!
+
+## Onboarding yourself
+
+So we went out, collected, considered and reflected on the feedback all of you provided (thanks again for that!) and overhauled the entire thing!
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:24]:
```suggestion
So we went and collected, considered, and reflected on the feedback you all provided (thanks again for that!), and overhauled the entire thing!  
```

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Big announcement incoming: we have redesigned the Tilt experience! 🥳  
+Today we’re taking a look at our onboarding. Most of you who are reading this are probably already acquainted with Tilt— don’t worry, we’ve got something coming for you, too.
+
+Everyone knows: you can only make one first impression, better make it count! So with that in mind, we wanted to create an experience that is quick, easy and fun, even if you’re starting from zero. That goes for both yourself and the friends you want to introduce Tilt to!
+
+## Onboarding yourself
+
+So we went out, collected, considered and reflected on the feedback all of you provided (thanks again for that!) and overhauled the entire thing!  
+The new tutorial helps you to get Tilt up and running yourself. With just one command, you can quickly tour Tilt’s best features, and be inspired by all the possibilities to boost your own developer workflow.
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:25]:
```suggestion
The new tutorial helps you to get Tilt up and running with just one command, so you can quickly tour Tilt’s best features, and learn about all the possibilities to boost your developer workflow.  
```

File: blog/_posts/2021-11-08-new-tutorial.md (hunk 1 above)
Comment (Chimney42) [reply blog/_posts/2021-11-08-new-tutorial.md:17]: Tilt user experience?

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Big announcement incoming: we have redesigned the Tilt experience! 🥳  
+Today we’re taking a look at our onboarding. Most of you who are reading this are probably already acquainted with Tilt— don’t worry, we’ve got something coming for you, too.
+
+Everyone knows: you can only make one first impression, better make it count! So with that in mind, we wanted to create an experience that is quick, easy and fun, even if you’re starting from zero. That goes for both yourself and the friends you want to introduce Tilt to!
+
+## Onboarding yourself
+
+So we went out, collected, considered and reflected on the feedback all of you provided (thanks again for that!) and overhauled the entire thing!  
+The new tutorial helps you to get Tilt up and running yourself. With just one command, you can quickly tour Tilt’s best features, and be inspired by all the possibilities to boost your own developer workflow.  
+All you need to get started, is Docker, Tilt and the sample source code. You’ll be able to play around with an actual multiservice application and see Tilt in action. [Try it out yourself](https://docs.tilt.dev/tutorial/index.html) and let us know what you think.
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:26]:
```suggestion
All you need to get started is Docker, Tilt and the sample source code (maybe; see below). You’ll be able to play around with an actual microservice application and see Tilt in action. [Try it out yourself](https://docs.tilt.dev/tutorial/index.html) and let us know what you think!
```

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+If you’d rather learn how to write your first Tiltfile to start, the [old tutorial](https://docs.tilt.dev/tiltfile_authoring.html) is still up.
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:28]: "Why would I wanna do that?"

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+## Onboarding your friends
+
+Still reading? Excellent!  
+Next step: getting your colleagues and managers on board! You might have asked yourself before: How can I quickly show Tilt to my buddy, but on their machine?  
+To help you with that, we’ve created a self-contained demo. Just run `tilt demo` and Tilt will spin up an entire dev cluster for you with our sample application. From scratch!
+
+![Tilt demo, from running tilt demo to spinning up the entire web service](/assets/images/new-tutorial/tilt-demo.gif)
+
+
+Do you have an issue onboarding someone to Tilt that we haven’t addressed yet? Let us know!
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:41]:
```suggestion
Do you have any onboarding-related issues we haven’t addressed yet? Let us know!
```

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+All you need to get started, is Docker, Tilt and the sample source code. You’ll be able to play around with an actual multiservice application and see Tilt in action. [Try it out yourself](https://docs.tilt.dev/tutorial/index.html) and let us know what you think.
+
+If you’d rather learn how to write your first Tiltfile to start, the [old tutorial](https://docs.tilt.dev/tiltfile_authoring.html) is still up.
+
+![Screenshot of the new Tilt docs showing the Tutorial overview](/assets/images/new-tutorial/docs-tutorial.png)
+
+## Onboarding your friends
+
+Still reading? Excellent!  
+Next step: getting your colleagues and managers on board! You might have asked yourself before: How can I quickly show Tilt to my buddy, but on their machine?
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:35]: One issue that often gets in the way of showing Tilt to one's colleagues is the non-Tilt parts of the setup e.g. setting up a Kubernetes cluster isn't for the faint of heart.

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+If you’d rather learn how to write your first Tiltfile to start, the [old tutorial](https://docs.tilt.dev/tiltfile_authoring.html) is still up.
+
+![Screenshot of the new Tilt docs showing the Tutorial overview](/assets/images/new-tutorial/docs-tutorial.png)
+
+## Onboarding your friends
+
+Still reading? Excellent!  
+Next step: getting your colleagues and managers on board! You might have asked yourself before: How can I quickly show Tilt to my buddy, but on their machine?  
+To help you with that, we’ve created a self-contained demo. Just run `tilt demo` and Tilt will spin up an entire dev cluster for you with our sample application. From scratch!
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:36]: We do need Docker installed though, don't we? `From scratch!` makes it sound like you don't.

File: blog/_posts/2021-11-08-new-tutorial.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+So we went out, collected, considered and reflected on the feedback all of you provided (thanks again for that!) and overhauled the entire thing!  
+The new tutorial helps you to get Tilt up and running yourself. With just one command, you can quickly tour Tilt’s best features, and be inspired by all the possibilities to boost your own developer workflow.  
+All you need to get started, is Docker, Tilt and the sample source code. You’ll be able to play around with an actual multiservice application and see Tilt in action. [Try it out yourself](https://docs.tilt.dev/tutorial/index.html) and let us know what you think.
+
+If you’d rather learn how to write your first Tiltfile to start, the [old tutorial](https://docs.tilt.dev/tiltfile_authoring.html) is still up.
+
+![Screenshot of the new Tilt docs showing the Tutorial overview](/assets/images/new-tutorial/docs-tutorial.png)
+
+## Onboarding your friends
```
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:32]:
```suggestion
## And one more thing
```

File: blog/_posts/2021-11-08-new-tutorial.md (hunk 8 above)
Comment (ellenkorbes) [inline blog/_posts/2021-11-08-new-tutorial.md:35]:
```suggestion
One issue that often gets in the way of showing Tilt to one's colleagues is the non-Tilt parts of the setup e.g. setting up a Kubernetes cluster isn't for the faint of heart.
```

**Review by ellenkorbes — CHANGES_REQUESTED** [review]
I'm sorry, I need an adult to stop me when copy-editing 😬

**Review by ellenkorbes — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**